gulpfile.js

```diff
@@ -12,6 +12,7 @@ var zip = require('gulp-zip');
 var merge = require('merge2');
 var path = require('path');
 var package = require('./package.json');
+var umd = require('gulp-umd');
```

Comment:
Can you move this require right after the `var uglify = require('gulp-uglify');` one?